[1.33.0] Release notes (#425)
replicatedhq/kots.io: content/release-notes/1.33.0.md

---
date: 2021-03-05T10:30:14-08:00
linktitle: "1.33.0"
title: "1.33.0"
kubernetes: "1.17, 1.18, and 1.19"
---
{{<features>}}
* NFS support in snapshots
{{</features>}}

{{<changes>}}
* Added instrumentation for preflight usage.
* Reduced disk usage during airgap install. 
* Upgraded [Helm v3](/vendor/helm/using-helm-charts/) dependency from v3.2.4 to v3.5.2
{{</changes>}}

{{<fixes>}}
* Fixed a bug where boolean custom fields were not displayed properly on the [license page](/kotsadm/updating/license-updates/).

> * Show correct page title while on the [Cluster Management](/vendor/guides/ha-guide/#adding-remaining-nodes-to-cluster) page. 

Reviewer: This should probably follow the "Fixed a bug..." format.

`Fixed a bug where the wrong page title was shown on the Cluster Management page`

* Removed duplicate secrets while generating [support bundle](/troubleshooting/support-bundle/).
* Fixed a bug that caused hidden groups to be shown on the [config page](/kotsadm/installing/online-install/#config-screen).

> * Fixed a bug that caused documents getting ignored in the Helm file

Reviewer 2: ```suggestion
* Fixed a bug that could cause documents to get ignored in Helm Charts
```
